server/src/modules-resolver.ts

```diff
@@ -5,20 +5,27 @@
  * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
  */
 
-import * as fs from 'fs';
-import * as paths from 'path';
+import * as fs from "fs";
+import * as paths from "path";
```

Comment:
Please don't change the quote types. Especially as the code base is predominantly using single quotes.